addons/vagrant/inventory/group_vars/wireless/hostapd.yml

```
@@ -3,42 +3,46 @@
 hostapd__configs:
   - name: open
     hostapd__acct_server_addr: '{{ packetfence_server_mgmt_ip }}'
-    hostapd__acct_server_port: '1813'
+    hostapd__acct_server_port: 1813
     hostapd__acct_server_shared_secret: 'anHardSharedSecret'
-    hostapd__auth_algs: '3'
+    hostapd__auth_algs: 3
     hostapd__auth_server_addr: '{{ packetfence_server_mgmt_ip }}'
-    hostapd__auth_server_port: '1812'
+    hostapd__auth_server_port: 1812
     hostapd__auth_server_shared_secret: 'anHardSharedSecret'
     hostapd__ctrl_interface: '/var/run/hostapd'
-    hostapd__ctrl_interface_group: '0'
+    hostapd__ctrl_interface_group: 0
     hostapd__driver: 'nl80211'
     hostapd__hw_mode: 'g'
     hostapd__interface: 'wlan0'
-    hostapd__logger_syslog: '1'
-    hostapd__macaddr_acl: '2'
+    hostapd__logger_syslog: 1
+    hostapd__macaddr_acl: 2
     hostapd__nas_identifier: 'HostAP-PF'
```

```suggestion
    hostapd__nas_identifier: '{{ ansible_fqdn }}'
```
That's way, we know request is coming from `wireless01`.